app/store/modules/session/index.js

```diff
@@ -1,9 +1,42 @@
-// store/modules/session/index.js
+import userData from './mock/user';
 
 export default {
   namespaced: true,
-  state: {},
-  getters: {},
-  mutations: {},
+  state: {
+    /**
+     * @type {Object}   Stores user data
+     */
+    user: userData, // TODO - remove mock data once the backend is available
+  },
+  getters: {
+    /**
+     * Gets a user.
+     *
+     * @param   {Object}  state   Current state
+     *
+     * @returns  {Object}  product   User data
+     */
+    user: state => state.user,
+
+    /**
+     * Check if a user is logged in.
+     *
+     * @param   {Object}  state   Current state
+     *
+     * @returns  {Object}  userLoggedIn   True if user is logged in, false otherwise
```

Comment:
- [ ] Type is currently `string` but according to description and method name should be `boolean`.